apps/recnet/src/components/rec/NewArticleForm.tsx

```diff
@@ -256,6 +256,7 @@ export function NewArticleForm(props: {
                     author: res.data.author,
                     year: res.data.year,
                     month: res.data.month ?? null,
+                    abstract: null,
```

Comment:
I know you fix here bcz of typechecking error. I noticed in `update.rec.dto.ts` and `create.rec.dto.ts`, we actually don't need `abstract` in the API payload. I think instead of adding a `abstract: null` here to satisfy type, what we need to do is to fix the API payload type in `recnet-api-model/src/lib/api/rec`.

```ts
export const recFormSubmissionSchema = z.intersection(
  z.union([
    z.object({
      articleId: z.string(),
      article: z.null(),
    }),
    z.object({
      articleId: z.null(),
      article: articleSchema.omit({
        id: true,
        isVerified: true,
        abstract: true, // ADD THIS
      }),
    }),
  ]),
  z.object({
    description: z.string(),
    isSelfRec: z.boolean(),
  })
);
```